### PR TITLE
fibjs 0.36.0

### PR DESCRIPTION
**Pull request by xicilion**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/fibjs.rb
Comment (carlocab) [inline Formula/fibjs.rb:5]: Please update the checksum; don't just remove it.

**Review by carlocab — CHANGES_REQUESTED** [review]

**Comment by xicilion** [comment]
Can we replace ubuntu in the CI with 20.04, or just skip the CI for Linux?

**Comment by xicilion** [comment]
There is also another way to implement docker compilation by specifying linux parameters to the build script.
The shell script looks like this:
```
bash build release linux
```
But I don't know how to change the rb.

**Comment by xicilion** [comment]
When will this request be merged, please?

**Comment by SMillerDev** [comment]
When it stops modifying the bottle block for one

**Comment by xicilion** [comment]
What does that mean, does it mean I need to manually fill the checksum generated by the ci back into the bottle?

**Comment by SMillerDev** [comment]
No, it means you should not modify the bottle block manually. And we won't merge the review as long as it does.

**Comment by xicilion** [comment]
If so, I want to remove the linux bottle and add the arm bottle, how do I do that?

**Comment by SMillerDev** [comment]
> If so, I want to remove the linux bottle and add the arm bottle, how do I do that?

You don't, if it builds and tests successfully there will be a bottle

**Comment by SMillerDev** [comment]
It seems to be building a bunch of vendored versions of Homebrew formulae, it really shouldn't do that.

**Comment by xicilion** [comment]
Finally, I changed the compilation options to pass the CI for Linux.

**Comment by xicilion** [comment]
Please review this pr to see if it can be merged.

**Review by SMillerDev — DISMISSED** [review]

**Comment by xicilion** [comment]
still not merged.

**Review by carlocab — DISMISSED** [review]

**Comment by xicilion** [comment]
Thanks a lot.

**Comment by carlocab** [comment]
Huh, guess this needs another review since I pushed to this.

**Comment by carlocab** [comment]
Merge workflow doesn't like my earlier review requesting changes even if I later approved. Will need to fix that.

**Pull request closed without merging**
